Feat(anta_runner): Add an option to enable running custom user tests (#6203)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

+  anta_report_exclude_statuses: [ "success" ]
```

> ### The execution of user-defined tests, located in `user_catalogs` by default, or as defined by `user_catalogs_dir_name` is now disabled by default

Reviewer: ```suggestion
### Execution of user-defined tests, stored in `user_catalogs` by default, or under the directory specified by `user_catalogs_dir_name` is now disabled by default
```

Author: Updated. Thanks!!

> Explicitly set `user_catalogs_enabled: true` to execute custom tests.

Reviewer: ```suggestion
Enable the execution of custom tests by explicitly setting `user_catalogs_enabled: true`.
```

Author: Updated.

> To execute your custom tests, you must explicitly set the `user_catalogs_enabled` to true

Reviewer 2: ```suggestion
Explicitly set `user_catalogs_enabled: true` to execute custom tests.
```

Author: updated. Thanks!!

> To execute your custom tests, you must explicitly set the `user_catalogs_enabled` to true

Reviewer 2: Please mention we need to set `user_catalog_dir` as well.

+       validate_bgp_peers: true
```

### Execution of user-defined tests, stored in `user_catalogs` by default, or under the directory specified by `user_catalogs_dir_name` is now disabled by default

> 

Reviewer 3: ```suggestion
### Execution of user-defined catalogs stored in `user_catalogs_dir` is now disabled by default

```

Author: Updated. Thanks!!

+       validate_bgp_peers: true
```

### Execution of user-defined catalogs stored in `user_catalogs_dir` is now disabled by default

Enable the execution of custom tests by explicitly setting `user_catalogs_enabled: true`.

```diff
name: ANTA Run
ansible.builtin.import_role:
  name: arista.avd.anta_runner
vars:
+  user_catalogs_enabled: true

> ```

Reviewer 3: ```suggestion
```diff
  - name: Run ANTA
    hosts: FABRIC
    connection: local
    gather_facts: false
    tasks:
      - name: Run ANTA on EOS devices
        import_role:
          name: arista.avd.anta_runner
      vars:
+      user_catalogs_enabled: true
```
```
